src/main/java/hello/CompileTest.kt

```diff
@@ -56,21 +64,26 @@ class KotlinScriptParser {
                 return hasErrors
             }
 
-            override fun report(severity: CompilerMessageSeverity, message: String, location: CompilerMessageLocation) {
-                val path = location.path
-                val position = if (path == null) "" else "$path: (${location.line}, ${location.column}) "
+            override fun report(severity: CompilerMessageSeverity, message: String, location: CompilerMessageLocation?) {
+                val path = location!!.path
+                val position = "$path: (${location.line}, ${location.column}) "
 
                 val text = position + message
```

Comment:
location can be null here. So it's be better to make something like:

```java
                val text = if (location != null) {
                    val path = location.path
                    val position = "$path: (${location.line}, ${location.column}) "
                    position + message
                } else {
                    message
                }
```

Reply:
You are right. I overlooked that while checking before pushing.
Thanks!